aw_watcher_window/macos.swift

```diff
@@ -225,13 +299,26 @@ class MainThing {
         data.url = activeTab.URL
         if let title = activeTab.title { data.title = title }
```

Comment:
This check needs to not only `if let title = activeTab.title { ... }`, but also if the string is empty, to prevent that bad permissions to lead to empty titles (I assume).

Is there a reason to do title assign here at all? Won't it just be the same?

Reply:
Not sure if there is a reason we don't just use the original title. Make a change locally to log if there is a diff and will report back.

Reply:
The title is different. Reason is the tab title is different than the window title in many cases. I've updated the code to conditionally use the tab title if it is not blank. Running this locally for a week and then will report back.

Reply:
Can you provide a short example of a differing title? Curious what kind of differences there are